docs/api/1.x/quotas.rst

```diff
@@ -0,0 +1,93 @@
+.. _api-quotas:
+
+Quotas management
+#################
+
+When the built-in plugin ``kinto.plugins.quotas`` is enabled in
+configuration, it becomes possible to configure quotas for a bucket or
+a collection.
+
+Clients can check for the ``quotas`` capability in the
+:ref:`root URL endpoint <api-utilities>`.
+
+.. note::
+
+    In terms of performance, enabling this plugin generates two or
+    three additional queries on backends per request.
+
+* A bucket's quota is a limit on the size of bucket attributes, group
+  attributes, collection attributes, and record attributes.
+* Deleted items are considered to have a size zero so if you add something
+  and remove it, it will look like it was never created for the
+  quota even if its tombstone is still there.
+* The quota plugin only works with the transactional storage backends
+  (i.e PostgreSQL)
+
+
+Configuration
+=============
+
+You can configure three types of quotas:
+
+* **QUOTA_BYTES**: The maximum total amount (in bytes) of data that
+  can be stored in sync storage, as measured by the Canonical JSON
+  stringification of every value plus every key's length.
+* **QUOTA_BYTES_PER_ITEM**: The maximum size (in bytes) of each
+  individual item in sync storage, as measured by the Canonical JSON
+  stringification of its value plus its key length.
+* **MAX_ITEMS**: The maximum number of items that can be stored in
+  a collection or bucket.
```

Comment:
Those upercase constants look weird (taken from storage.sync isn't it ?) Why don't they all start with QUOTA ? Why using them after all ?